drivers/overlay/overlay.go

```diff
@@ -431,6 +441,20 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return "", err
 	}
 
+	if _, err = os.Stat(string(lowers)); err != nil {
```

Comment:
If the file contains a colon-separated list of lower link partial paths, don't we need to be splitting this list?